### ISSUE TITLE
Failed to install, cannot find org.nixos.darwin-store

### ISSUE THREAD

**Issue by ciniglio** [DeterminateSystems/nix-installer#1097]
## Error
```
Error: 
   0: Install failure
   1: Error executing action
   2: Action `create_nix_volume` errored
   3: Action `kickstart_launchctl_service` errored
   4: Failed to execute command with status 113 `"launchctl" "kickstart" "-k" "system/org.nixos.darwin-store"`, stdout: 
      stderr: Could not find service "org.nixos.darwin-store" in domain for system


```

## Metadata
|key|value|
|--|--|
|**version**|0.20.2|
|**os**|macos|
|**arch**|aarch64|

**Comment by cole-h** [comment]
Did you have Nix installed at some point in the past? Could you see if `/Library/LaunchDaemons/org.nixos.darwin-store.plist` exists, and if it does, its contents? Could you also see if `darwin-store` appears in the output of `sudo launchctl print-disabled system`?

**Comment by ciniglio** [comment]
I did have it installed; I purged a bunch of things and got it working eventually. Unfortunately, I don't remember which exact thing I removed to make it past this error. Most of the other errors in the installer log were very helpful (e.g. remove things from fstab or disk utility or delete a user), but this error was not as clear on what I should remove.

**Comment by cole-h** [comment]
If you run into this again and can provide more information, I'd be glad to debug further! But for now I am going to close this. Thanks!

**Comment by 20k-ultra** [comment]
I have the same scenario on my machine. I had nix installed then removed it to run this repo's setup. I removed the previous nix installation as best as one can since it's not a straight forward single command. I followed https://iohk.zendesk.com/hc/en-us/articles/4415830650265-Uninstall-nix-on-MacOS though.

> Could you see if /Library/LaunchDaemons/org.nixos.darwin-store.plist exists

This file does not exist.

> Could you also see if darwin-store appears in the output of sudo launchctl print-disabled system?

darwin-store does not appear in the output of this command.
